src-vue/src/views/SettingsView.vue

```diff
@@ -78,9 +78,10 @@ export default defineComponent({
             get(): boolean {
                 return this.$store.state.enableReleasesSwitch;
             },
-            set(value: boolean): void {
+            async set(value: boolean): Promise<void> {
```

Comment:
Uh, the `Promise` return value was introduced by running a quick fix action in vscode. Not sure if correct ^^"

Reply:
Yeah it's correct